### ISSUE TITLE
Error when launching bot.

### ISSUE THREAD

**Issue by Kru0489** [cusdt-LINK/kcc-flashswap-arb#4]
I was able to deploy the smart contract without issue, but I'm getting the following error when I try to run the bot. Thanks in advance.
TSError: ⨯ Unable to compile TypeScript:
bot/index.ts:63:13 - error TS2571: Object is of type 'unknown'.

63         if (err.message === 'Too much pending tasks' || err.message === 'async-lock timed out') {
               ~~~
bot/index.ts:63:57 - error TS2571: Object is of type 'unknown'.

63         if (err.message === 'Too much pending tasks' || err.message === 'async-lock timed out') {
                                                           ~~~

**Comment by alexbobes** [comment]
I've encountered the same error; commented the if statement and it works:
...
      } catch (err) {
        // if (err.message === 'Too much pending tasks' || err.message === 'async-lock timed out') {
        //   return;
        // }
        log.error(err);
      }
...

**Comment by Kru0489** [comment]
Great thanks! Any input on the "cannot estimate gas; transaction may fail or require manual gas limit" error?

**Comment by gweidart** [comment]
> I was able to deploy the smart contract without issue, but I'm getting the following error when I try to run the bot. Thanks in advance. TSError: ⨯ Unable to compile TypeScript: bot/index.ts:63:13 - error TS2571: Object is of type 'unknown'.
> 
> 63 if (err.message === 'Too much pending tasks' || err.message === 'async-lock timed out') { ~~~ bot/index.ts:63:57 - error TS2571: Object is of type 'unknown'.
> 
> 63 if (err.message === 'Too much pending tasks' || err.message === 'async-lock timed out') { ~~~

Fixed here - #2

**Comment by gweidart** [comment]
> Great thanks! Any input on the "cannot estimate gas; transaction may fail or require manual gas limit" error?

Fixed here - #3
